samples/CSharp/SourceGenerators/GeneratedDemo/GeneratedDemo.csproj

```diff
@@ -7,16 +7,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <AdditionalFiles Include="MainSettings.xmlsettings" />
-    <None Include="MainSettings.xmlsettings">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
+    <AdditionalFiles Include="People.csv" CsvLoadType="Startup" />
+    <AdditionalFiles Include="Cars.csv" CsvLoadType="OnDemand" CacheObjects="true" />
+
+    <AdditionalFiles Include="MainSettings.xmlsettings" CopyToOutputDirectory="Always" />
+    <None Include="MainSettings.xmlsettings" CopyToOutputDirectory="Always" /> <!-- TODO: remove this when AdditionalFiles supports CopyToOutputDirectory -->
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\SourceGeneratorSamples\SourceGeneratorSamples.csproj" 
-                      OutputItemType="Analyzer"
-                      ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\SourceGeneratorSamples\SourceGeneratorSamples.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="true" />
```

Comment:
```suggestion
    <ProjectReference Include="..\SourceGeneratorSamples\SourceGeneratorSamples.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Reply:
We're using the attributes from the assembly, so in this case we need to reference the output assembly too. We could split the dependencies out into generators and runtime assemblies.

Reply:
The previous approach (generating the attribute) seemed like a good approach. Is there a reason we cannot do that?